LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2Ability_WeaponAbilities.uc

```diff
@@ -0,0 +1,123 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    X2Ability_SMGAbilities.uc
+//  AUTHOR:  Amineri (Long War Studios)
+//  PURPOSE: Adds stat modifier abilities for all tech tiers of SMG
+//           
+//---------------------------------------------------------------------------------------
+class X2Ability_WeaponAbilities extends X2Ability config(WeaponAbilities);

```

Comment:
Could you please rename to something more specific? Perhaps `X2Ability_BullpupAbilities` or `X2Ability_FactionWeaponAbilities`?

Reply:
And yes, I'd prefer it if the config loaded from `GameData_WeaponData`.

Reply:
Ok I will rename it to X2Ability_FactionWeaponAbilities.

Reply:
This has been renamed.